python/smqtk/algorithms/saliency/sbsm.py

```diff
@@ -0,0 +1,208 @@
+import numpy as np
+from tqdm import tqdm
+import copy
+from PIL import Image
+import os
+from sklearn.metrics.pairwise import euclidean_distances
+import six
+from smqtk.algorithms import SmqtkAlgorithm
+from smqtk.algorithms.saliency import SaliencyBlackbox,ImageSaliencyAugmenter
+from smqtk.representation.data_element.memory_element import DataMemoryElement
+
```

Comment:
There should be two new-lines between imports and the first class/function definition.

From https://www.python.org/dev/peps/pep-0008/#blank-lines:
```
Surround top-level function and class definitions with two blank lines.
```

Reply:
How about now?